modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentLauncher.java

```diff
@@ -375,13 +374,14 @@ public static void main(String[] args) throws Exception {
                     builder.connectionSpecs(sslConnectionSpec(cipherSuites));
             }
 
-            OkHttpClient sslFactory = builder.build();
-
-            opts.callFactory = sslFactory;
-            opts.webSocketFactory = sslFactory;
             opts.secure = true;
         }
 
+        OkHttpClient okHttpClient = builder.build();
+        
```

Comment:
It seems that after this change SSL will not work.

Reply:
All works fine. This was incorrect name of variable in original code.
`OkHttpClient sslFactory = builder.build()`